validation/src/static/scripts/index.ts

```diff
@@ -1,27 +1,28 @@
 // Helper function for shorter 'document.getElementById(id)'
 const getElem = (id: string) => document.getElementById(id);
 
+const noElemError = new Error(
+  'An Element essential for UI functionality could not be found.'
+);
+
```

Comment:
I think it's better to throw at the place of failure, so the line number + stack trace are correct. I'd suggest just defining the error message up here and throwing it where it happens :)
![image](https://user-images.githubusercontent.com/14766017/91313708-f85a8480-e7b5-11ea-8eee-49622990d69d.png)

Reply:
Isn't that what I've done?